src/main/kotlin/com/cognifide/gradle/aem/common/mvn/MvnGav.kt

```diff
@@ -1,6 +1,7 @@
 package com.cognifide.gradle.aem.common.mvn
 
 import groovy.util.*
+import groovy.xml.XmlParser
```

Comment:
I was thinking that it is deprecated...

Reply:
No, `groovy.util.XmlParser` is deprecated.

Reply:
ok thx clear